src/ui/dictionarybar.cc

```diff
@@ -95,7 +95,7 @@ void DictionaryBar::setDictionaryIconSize( IconSize size )
 
     case IconSize::Large: {
       auto largeSize = QApplication::style()->pixelMetric( QStyle::PM_LargeIconSize );
-      setIconSize( { largeSize + 10, largeSize + 10 } );
+      setIconSize( { largeSize + 10, largeSize + 10 } ); // the value isn't large enough, so we add 10
```

Comment:
How about we just set the large icon size as 2x or 1.5x or 1.25x of normal, because a theme can choose whatever is “large”.

For example, A theme can decide normal is 36 while large is 40, but `normal x 1.5` or `normal x 2` is guaranteed to be “large”.

Reply:
i think the QStyle::PM_LargeIconSize technique was already good and felt correct, there's no problem in adding 10 , right?

multiplying the "normal" size feels a bit weird